lib/chainweb/p2p/block_branches_bounds_request_body.ex

```diff
@@ -1,6 +1,6 @@
-defmodule Kadena.Chainweb.P2P.BlockHashRequestBody do
+defmodule Kadena.Chainweb.P2P.BlockBranchesBoundsRequestBody do
```

Comment:
`BlockBranchesBoundsRequestBody` seems like a very long name. How about renaming it to `BlockBranchesRequestBody`?

The "branches" already make reference to the lower and upper bounds required. So the "bounds" could be removed from the module name.